### PR TITLE
Mac-conda-plugin and other docs

### PR DESCRIPTION
**Pull request by loriab**
## Description
Accumulated docs improvements. Update samples.

## Todos
Notable points that this PR has either accomplished or will accomplish.
* **Developer Interest**
  - [x] Add download/install notes for Windows
  - [x] Update header and doc citation to JCTC article. Add licensing blurb.
  - [x] Fix and document building plugins from conda on Mac
  - [x] Make X2C input and output consistent. @fevangelista, is this what you wanted?
  - [x] Fix `basisset_ = basisset_;` typo noticed by CDS
* **User-Facing for Release Notes**

## Status
- [x] Ready to go **Everyone**, stop review when you hit "samples/"

### CODE REVIEW

File: doc/sphinxman/source/build_planning.rst [hunk 1]
```diff
@@ -1250,7 +1250,7 @@ commonly set up through the :envvar:`PSI_SCRATCH` environment variable:
 
 .. code-block:: bash
 
-    # sh, bash: add to shell or ~/.bashrc (Linux) or ~/.bash_profile (Mac) file
+    # sh, bash: add to shell or ~/.bashrc (Linux/Windows) or ~/.bash_profile (Mac) file
```
Comment (dgasmith) [inline doc/sphinxman/source/build_planning.rst]: Never thought I would see the day.

File: doc/sphinxman/source/conda.rst [hunk 2]
```diff
@@ -75,7 +76,7 @@ How to install a Psi4 binary with the Psi4conda installer, download site
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 `Download one of the six installers
```
Comment (dgasmith) [inline doc/sphinxman/source/conda.rst]: Reads a bit strange, can we say 9 even though its just six?

File: doc/sphinxman/source/introduction.rst [hunk 3]
```diff
@@ -93,6 +93,19 @@ Overall |PSIfour| Package
 The following citation should be used in any publication utilizing the
 |PSIfour| program package:
 
+* "Psi4 1.1: An Open-Source Electronic Structure Program Emphasizing
+  Automation, Advanced Libraries, and Interoperability", R. M. Parrish, L.
```
Comment (andysim) [inline doc/sphinxman/source/introduction.rst]: I haven't checked how this renders, but does the break in "L. A. Burns" cause it to be split across two lines?  If so, you probably want a line break before the L.  LaTeX allows things to be forced onto the same line with non-breaking spaces (~), but I'm not aware of a simlar feature in REST.

**Comment by loriab** [comment]
@andysim, it renders ok, I believe. screen shot 2017-05-13 at 1 13 52 pm

File: doc/sphinxman/source/conda.rst (hunk 2 above)
Comment (loriab) [inline doc/sphinxman/source/conda.rst]: good point. i'll fix it in the rebase after Libint.

File: doc/sphinxman/source/build_planning.rst (hunk 1 above)
Comment (loriab) [inline doc/sphinxman/source/build_planning.rst]: Indeed. But David tried it the night before FAME and it just worked (increasing font size for projection seems to be the only missing piece), so I figured it was time to acknowledge.

**Pull request merged**